packages/vite/src/index.ts

```diff
@@ -0,0 +1,170 @@
+import { readFile as fsReadFile, existsSync } from 'fs'
+import path from 'path'
+import { promisify } from 'util'
+
+import { NodeGlobalsPolyfillPlugin } from '@esbuild-plugins/node-globals-polyfill'
+import { viteCommonjs } from '@originjs/vite-plugin-commonjs'
+import react from '@vitejs/plugin-react'
+import { transform } from 'esbuild'
+import { normalizePath, UserConfig } from 'vite'
+import { createHtmlPlugin } from 'vite-plugin-html'
+
+import { getWebSideDefaultBabelConfig } from '@redwoodjs/internal/dist/build/babel/web'
+import { getConfig } from '@redwoodjs/internal/dist/config'
+import { getPaths } from '@redwoodjs/internal/dist/paths'
+
+const redwoodConfig = getConfig()
+const redwoodPaths = getPaths()
```

Comment:
These functions are executed as soon as this file is imported, which may not be a problem given the changes I've made to the CLI to ensure the `cwd` is set before the CLI imports anything. But just something to note if you think this file could be executed somehow before `cwd` is set

Reply:
Hmmm... I'll move this inside the function!